packages/typespec-test/test/openai/generated/typespec-ts/src/outputModels.ts

```diff
@@ -9,8 +9,6 @@ export interface DeploymentOutput {
   readonly deploymentId: string;
 }
 
-export interface AzureCoreTraitsOutput {}
```

Comment:
what is causing the change here?

Reply:
this is somehow introduced by our template logic and i remove that.